### PR TITLE
Updated the documentation list with a events section. For starters

### PR DESCRIPTION
**Pull request by adnasa**

### CODE REVIEW

File: examples/documentation.html [hunk 1]
```diff
@@ -95,25 +106,43 @@
...
+                      propdom.append($('<span class="label">Events: </span>'));
+                      var dispatchString = dispatchList.map(function(dispatcherName) {
+                        return '<span class="event">' + dispatcherName + '</span>';
+                      }).join(' ');
+                      propdom.append(dispatchString);
+                    }
+
+                    if (propdom.children().length) {
+                      propdom.addClass('populated');
+                      c.append(propdom);
```
Comment (liquidpele) [inline examples/documentation.html]: All your indention in added code is 2 spaces... should be 4 to match the rest of the code.

**Comment by liquidpele** [comment]
This is a great idea!

File: examples/documentation.html (hunk 1 above)
Comment (adnasa) [inline examples/documentation.html]: Will fix that now ;)

**Comment by adnasa** [comment]
it may be a good idea to setup an editorconfig ;)

File: examples/documentation.html [hunk 2]
```diff
@@ -38,6 +38,17 @@
             container.append(op);
         }
 
+        function getProps(obj, key) {
+            var props = [];
+            if (obj.hasOwnProperty(key)) {
+              var _obj = obj[key];
+              if (typeof _obj === 'object') {
+                props.push.apply(props, Object.keys(_obj));
+              }
```
Comment (liquidpele) [inline examples/documentation.html]: still only 2 spaces under this if block...

File: examples/documentation.html [hunk 3]
```diff
@@ -95,24 +106,42 @@
...
                         propdom.append($(pstrings.join(', ')));
+                    }
+
+                    // add event dispatchers by the chart;
+                    dispatchList = getProps(chart, 'dispatch');
+                    if (dispatchList.length) {
+                        propdom.append($('<br/>'));
+                        propdom.append($('<span class="label">Events: </span>'));
+                        var dispatchString = dispatchList.map(function(dispatcherName) {
+                          return '<span class="event">' + dispatcherName + '</span>';
```
Comment (liquidpele) [inline examples/documentation.html]: only 2 spaces here

**Pull request merged**